Tests/Tests/STPPaymentContextTest.m

```diff
@@ -0,0 +1,97 @@
+//
+//  STPPaymentContextTest.m
+//  Stripe
+//
+//  Created by Brian Dorfman on 8/1/16.
+//  Copyright © 2016 Stripe, Inc. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import "STPAPIClient.h"
+#import "STPPaymentContext.h"
+
+@interface STPPaymentContext (Testing)
+- (PKPaymentRequest *)buildPaymentRequest;
+@end
+
+@interface TestSTPBackendAPIAdapter: NSObject <STPBackendAPIAdapter>
```

Comment:
nit: can we move this to its own file? i imagine we'll want to expand this in the future, and may want it in other test files.